nms/app/packages/magmalte/app/components/lte/LteContext.js

```diff
@@ -96,6 +96,12 @@ export function GatewayContextProvider(props: Props) {
       setIsLoading(false);
     };
     fetchState();
+    // Refresh data every 30 seconds
+    const interval = setInterval(() => {
```

Comment:
Instead of hardcoding the time here, can you use a variable here, we can perhaps move this to a configurable settings component later on. 

Additionally I was wondering if we can trigger this more explicitly through a context consumer, rather than autorefreshing by default. Thanks